index.md

```diff
@@ -1,47 +1,42 @@
 
-## Sysdig Chart Repository
+## Core Component Charts
 
-The Sysdig Chart Repository includes the following charts:
+To install the core components of Sysdig Monitor and Sysdig Secure, use the **sysdig-deploy** parent chart, which invokes a variety of child charts as needed. 
 
-| Charts          | Description |
-| :---------------| ----------- |
-|[`sysdig-deploy`](/charts/sysdig-deploy)    | Deploys the following components in your Kubernetes cluster:<br>`kspm-collector`:  Enables collecting and sendin Kubernetes resource manifests to be evaluated against compliance policies. The scan results are displayed by Sysdig Secure's Actionable Compliance.<br>`node-analyzer`: Enables  Image Analyzer, Benchmarks, and Host Scanning features.<br>`rapid-response`:  Allows designated advance users to investigate and troubleshoot events from a remote shell.<br>`agent`: Processes syscall events and metrics, creates [capture](https://docs.sysdig.com/en/docs/sysdig-secure/investigate/captures/#captures) files, and performs auditing and compliance. |
-|[`admission-controller`](/charts/admission-controller) | Deploys the [Sysdig Admission Controller](https://docs.sysdig.com/en/docs/sysdig-secure/scanning/admission-controller/) in your Kubernetes cluster to enable image scanning and audit logging. |
-|[`cloud-connector`](/charts/cloud-connector) | Deploys the [Sysdig Cloud Connector](https://docs.sysdig.com/en/docs/installation/sysdig-secure-for-cloud/) in your Kubernetes cluster to enable threat-detection and image scanning. |
-|[`harbor-scanner-sysdig-secure`](/charts/harbor-scanner-sysdig-secure) | Deploys the Harbor Scanner Adapter to enable Harbor to use Sysdig Secure scanning engine to analyze the container images managed by the platform. |
-|[`registry-scanner`](/charts/registry-scanner) | Deploys the Sysdig Registry Scanner as a scheduled cronjob in your Kubernetes cluster. |
-|[`sysdig-mcm-navmenu`](/charts/sysdig-mcm-navmenu) | Deploys services and ingress required to create Multi-Cloud Management menu in Sysdig. |
-|[`sysdig-stackdriver-bridge`](/charts/sysdig-stackdriver-bridge) | Deploys the the Sysdig Stackdriver Bridge application to enable reading and sending audit logs from Stackdriver to Sysdig agent. |
+Additional Sysdig Secure components provide scanning for vulnerabilities in various parts of the environment and the CI/CD lifecycle.
 
+| Parent Chart                                   | Subcharts                                              | Description                                                  |
+| ---------------------------------------------- | :----------------------------------------------------- | ------------------------------------------------------------ |
+| [`sysdig-deploy`](/charts/sysdig-deploy)       |                                                        | Installs, configures, and upgrades the Sysdig Agent and a variety of Sysdig Secure components. The parent chart deploys multiple child charts. |
```

Comment:
It looks like the `cluster-scanner` subchart has been missed here. It was added to `sysdig-deploy` a few weeks back and can be found [here](https://github.com/sysdiglabs/charts/tree/master/charts/cluster-scanner). I know the feature is in CA, so maybe this omission was intentional, but it's already included in the chart so I see no problem with documenting it here. @mavimo , do you have thoughts on this?

Reply:
Need advice from @mavimo -- till now we were told not to mention it

Reply:
@aroberts87 I personally suggest waiting until it goes GA. I know this is available in the codebase but this is more of a "documentation" page and perhaps it is worth listing the charts we support; if we want to add it IMHO we should state that it is in CA.

FYI JM2C, I'm not concerned too much on adding the `cluster-scanner` as well.

Reply:
Sounds good to me! Let's 🚢 it! 🙂